src/components/Assets/Buttons/Button.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <button :class="vclass" class="btn">{{value}}</button>
+  <button :class="vclass" class="btn" type="button">{{value}}</button>
```

Comment:
Słuszna zmiana.

Reply:
Jeśli button nie jest otoczony formularzem, to button bez typu jest zwykłym buttonem i nie jest mu potrzebny type aby funkcjonował jako button